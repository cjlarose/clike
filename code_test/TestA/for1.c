
void printint(int);

void main()
{
  int i,j;

  for (i=0; i<10; i = i + 1) {
     printint(i);
  };
}

