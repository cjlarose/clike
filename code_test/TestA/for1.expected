Loaded: /usr/local/share/spim/exceptions.s
0123456789